workers/unity/Packages/com.improbable.gdk.mobile/Android/Utility/AndroidDeviceInfo.cs

```diff
@@ -2,22 +2,22 @@
 
 namespace Improbable.Gdk.Mobile.Android
 {
+    public enum AndroidDeviceType
```

Comment:
Do we need the Android prefix, considering the class is already called `AndroidDeviceInfo`?

Reply:
nope

Reply:
The enum was taken out of AndroidDeviceInfo to allow using it directly (just AndroidDeviceType instead of AndroidDeviceInfo.DeviceType).